examples/pretrained_word_embeddings.py

```diff
@@ -105,15 +105,15 @@
 print('Preparing embedding matrix.')
 
 # prepare embedding matrix
-num_words = min(MAX_NB_WORDS, len(word_index))
+num_words = min(MAX_NB_WORDS, len(word_index) + 1)
```

Comment:
That's somewhat inaccurate, there would actually be at most `len(word_index)` words. But you are right that they are 1-indexed. I would put the `+ 1` on the line below, when instantiating the embedding matrix.